packages/plugin-doc-coverage/src/lib/config.unit.test.ts

```diff
@@ -0,0 +1,64 @@
+import { describe, expect, it } from 'vitest';
+import {
+  type DocCoveragePluginConfig,
+  docCoveragePluginConfigSchema,
+} from './config.js';
+
+describe('docCoveragePluginConfigSchema', () => {
+  it('accepts a documentation coverage configuration with all entities', () => {
+    expect(() =>
+      docCoveragePluginConfigSchema.parse({
+        language: 'typescript',
+        sourceGlob: 'src/**/*.{ts,tsx}',
+      } satisfies DocCoveragePluginConfig),
+    ).not.toThrow();
+  });
```

Comment:
```suggestion
it('accepts a full documentation coverage configuration object', () => {
    expect(
      docCoveragePluginConfigSchema.parse({
        language: 'typescript',
        sourceGlob: 'src/**/*.{ts,tsx}',
        outputFolderPath: './docs',
      } satisfies DocCoveragePluginConfig),
    ).toStrictEqual({
      language: 'typescript',
      sourceGlob: 'src/**/*.{ts,tsx}',
      outputFolderPath: './docs',
    });
  });
```

The first example is a nice overview to learn default usage or functionality of the code. 
Therefor I tried to add all possible options and also used `toStrictEqual` so the reader can see there is no transformation or defaults included in the parsing.